pandas/plotting/api.py

```diff
@@ -0,0 +1,20 @@
+"""
+Plotting api
+"""
+
+# flake8: noqa
+
+try:  # mpl optional
+    from pandas.plotting import converter as conv
```

Comment:
I would leave out the `conv` (now both `conv` and `converter` in pandas.plotting namespace)